src/20_Components/amp-gist.html

```diff
@@ -0,0 +1,42 @@
+<!--
+  ## Introduction
+
+  The amp-gist component allows embedding of an entire gist or a single file.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-gist/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!-- Import the amp-gist component -->
+  <script async custom-element="amp-gist" src="https://cdn.ampproject.org/v0/amp-gist-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <!---{
+    "experiments": ["amp-gist"]
+  }--->
+</head>
+<body>
+
+  <!-- ## Basic Usage -->
+  <!-- Embed an entire gist. -->
+  <amp-gist
+    data-gistid="b9bb35bc68df68259af94430f012425f"
+    layout="responsive"
```

Comment:
+1 to what John said. I'd be fine with keeping it as it is, but making it explicit in the docs that:

* fixed-height layout should be used
* providing an accurate height is important to avoid jumps (obtain height e.g. via Chrome Dev Tools on gist page - gists are usually used by devs, so I think it's fine to recommend Dev Tools)